db/schema.rb

```diff
@@ -184,7 +184,7 @@
   create_table "vacancies", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "job_title", null: false
     t.string "slug", null: false
-    t.text "job_summary", null: false
+    t.text "job_summary"
```

Comment:
In what scenario will job_summary be null?

Reply:
When a user completes step 1 of the create a job journey (e.g every time a vacancy is created)